app/src/main/res/values/strings.xml

```diff
@@ -430,8 +430,16 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="additional_contexts_and_projects">Additional contexts and projects</string>
     <string name="enter_space_seperated_contexts_and_projects_these_will">Enter space-separated @contexts and +projects. These will always be visible at todo.txt insert dialogs.</string>
     <string name="infinite">Infinite</string>
-    <string name="search_project">Search projects</string>
-    <string name="search_context">Search contexts</string>
+    <string name="filter_by_project">Filter by project</string>
+    <string name="filter_by_context">Filter by context</string>
     <string name="insert_project">Insert projects</string>
     <string name="insert_context">Insert contexts</string>
+    <string name="filter_todo">Filter todo</string>
+    <string name="filter_by_priority">Filter by priority</string>
+    <string name="filter_by_due_date">Filter by due date</string>
+    <string name="due_overdue">Overdue</string>
+    <string name="due_today">Due today</string>
+    <string name="due_future">Due in future</string>
```

Comment:
how about "Due %s"with   localizedDate

Reply:
This is to minimize strings for translation etc?

Then maybe `Due %c today` where %c can be >, =, <?

Reply:
overdue and today is ok. But for in future I'd suggest to just append the date 
name="due">Due

+= dateformat(dt)

Reply:
Due today might be also combinable.

But overdue and "due x" might be completely different in a few languages

Reply:
Not sure I understand what / if you would like me to change.

Initially, my thinking was that making these 3 separate strings would maximize translatability.